exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -81,6 +83,11 @@
       new RangeLongValidator("planner.identifier_max_length", 128 /* A minimum length is needed because option names are identifiers themselves */,
                               Integer.MAX_VALUE, DEFAULT_IDENTIFIER_MAX_LENGTH);
 
+  public static final OptionValidator FILTER_MIN_SELECTIVITY_ESTIMATE_FACTOR = new MinRangeDoubleValidator("planner.filter.min_selectivity_estimate_factor",
```

Comment:
Thinking about this more, we should probably remove the term 'Filter' from the option.   Calcite's RelMdSelectivity has a default implementation for various Rels, not just Filter.  In the future we could potentially use the same min and max bounds for other Rels.    What do you think ?  If you agree, other places need to be modified too.